ui/pages/settings/info-tab/info-tab.test.tsx

```diff
@@ -45,9 +46,11 @@ describe('InfoTab', () => {
     it('should trigger support modal when click support link', () => {
       const supportLink = getByText('Visit our support center');
       fireEvent.click(supportLink);
-      expect(
-        getByTestId('visit-support-data-consent-modal'),
-      ).toBeInTheDocument();
+      waitFor(() =>
```

Comment:
It seems like `waitFor()` is async and returns a Promise, we should add an `await` before this